activesupport/lib/active_support/core_ext/module/attribute_accessors.rb

```diff
@@ -24,7 +24,7 @@ class Module
   #   end
   #   # => NameError: invalid attribute name: 1_Badname
   #
-  # If you want to opt out the creation on the instance reader method, pass
+  # If you want to opt out of the creation of the instance reader method, pass
```

Comment:
This phrasing reads strange. How about just “skip adding” in general?

Here that would be “To skip adding the instance reader method,”

Reply:
:+1: API docs are [supposed to avoid "you"](https://guides.rubyonrails.org/api_documentation_guidelines.html#wording), so this is an improvement on that front too.

Maybe consider "To omit"?

Reply:
"To skip adding the instance reader method, pass"
or
"To omit the creation of the instance reader method, pass"
or
"To omit the creating the instance reader method, pass"
?

Or something else?

Reply:
"To omit the instance reader method", I think.

Reply:
Thanks, I pushed the changes.